dss_datamover/nfs_cluster.py

```diff
@@ -41,7 +41,19 @@ class NFSCluster:
     manager = Manager()
 
     def __init__(self, config={}, user_id="ansible", password="password", logger=None):
-        self.config = config.get("nfs", {})
+        self.config = config.get("fs_config", {}).get("nfs", {})
+        self.nfs_port = config.get("fs_config", {}).get("nfsport", {})
+        # override with CLI args if applicable
+        nfs_cli_args = ['nfs_server', 'nfs_port', 'nfs_share']
+        if set(nfs_cli_args).issubset(config):
+            if config['nfs_server'] is not None and config['nfs_share'] is not None:
```

Comment:
`is not None` is not needed. Just `if config['nfs_server'] and config['nfs_share']` is good enough.